chunkator/__init__.py

```diff
@@ -11,9 +11,12 @@ def chunkator(queryset, chunk_size):
     length. We're saving at least a ``count()`` query on QuerySets, or a
     CPU-and-RAM-consuming ``len(queryset)`` query.
     """
-    pk = 0
+    pk = None
     while True:
-        page = queryset.filter(pk__gt=pk).order_by('pk')[:chunk_size]
+        queryset = queryset.order_by('pk')
```

Comment:
@rodo I was talking about records inserted **during** the export or dump. if we're using unordered (and/or) non-unique sorting keys, yes, we'll miss them.
If we're using sequential, we won't miss the record inserted 1ms *before* the last query.